common.proto

```diff
@@ -16,14 +16,36 @@ message ConfigurationFormResponse {
 }
 
 message FormField {
+  oneof field {
```

Comment:
Why proto file changes are present? Please rebase the branch before merging.

Reply:
fixed